package.json

```diff
@@ -1,13 +1,21 @@
 {
   "name": "bravado-core",
-  "version": "0.12.2",
+  "version": "0.13.0",
   "description": "REST (Level 3/HATEOAS) API Framework",
   "directories": {
     "lib": "lib"
   },
   "scripts": {
+    "reinstall": "rm -rf node_modules && yarn install",
+    "lint": "esw . --ext .js",
+    "lint:fix": "yarn lint --fix",
+    "lint:watch": "yarn lint --watch",
+    "lint:changed": "lint-staged",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
we should change this...maybe an intern or something could do something like this.